lib/src/tokenizer.cc

```diff
@@ -185,10 +185,16 @@ namespace hocon {
     }
 
     /** Characters JSON allows a number to start with */
-    static const string first_number_chars = "0123456789-";
+    static string first_number_chars() {
```

Comment:
This is done to avoid too much global static initialization (and some races we got into using static initialization for test variables). @peterhuene @kylog @branan have patterns you prefer here? Singletons could also be used, but I don't have a strong case for it.